packages/synapse-react-client/src/umd.index.ts

```diff
@@ -175,7 +175,8 @@ const SynapseComponents = {
 
 // Include the version in the build
 // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-member-access
-const SynapseReactClientVersion = require('../package.json').version
+// @ts-ignore
+const SynapseReactClientVersion = window._SRC_VERSION
```

Comment:
Since we are using ESM now, we can't use `require`. And `import`ing a JSON file dynamically is not going to work in our bundles. So use a global that we will set in the Vite config.